src/Audit/Drupal/FilesManaged.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Drutiny\Audit\Drupal;
+
+use Drutiny\Audit;
+use Drutiny\Sandbox\Sandbox;
+
+/**
+ * Anonymous sessions
+ */
+class FilesManaged extends Audit {
+
+  /**
+   *
+   */
+  public function audit(Sandbox $sandbox) {
+    $stat = $sandbox->drush(['format' => 'json'])->status();
+    $root = $stat['root'];
+    $files = $stat['files'];
+ 
+    $filesManaged = explode("\n",$sandbox->drush()->sqlq("SELECT uri FROM file_managed"));
```

Comment:
Unsure on how well this scales when you have tens of thousands of managed files. Do you have any performance metrics? Would it be worth doing a count first?

Reply:
What about multiple calls with LIMITed SQL?

Reply:
Unsure, do you have an example metrics on how long this takes to run, e.g. with 100 managed files, 10,000 etc? Ideally a check will not take more than a few seconds to run.

Reply:
"SELECT COUNT(*) FROM file_managed" => 578
Debug code:
   var_dump('Start:');
    var_dump(time());
    $filesManaged = explode("\n",$sandbox->drush()->sqlq("SELECT uri FROM file_managed"));
    var_dump('End:');
    var_dump(time());die;
Timing:
string(6) "Start:"
int(1509960808)
string(4) "End:"
int(1509960810)

Reply:
also 2 seconds for 2691 files in DB

Reply:
That is cool.